manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient

```

Comment:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```
There's already a folder named `FACEITLTD` (`manifests/f/FACEITLTD`) in the repository; so update it to match that.